pkg/run/bootstrap/bootstrap_wizard_ui.go

```diff
@@ -354,21 +446,24 @@ func (m wizardModel) getContent() string {
 
 	b.WriteString("Please enter the following values" + "\n" +
 		"(Tab and Shift+Tab to move input selection," + "\n" +
+		"(Space to toggle the currently focused checkbox," + "\n" +
 		"Enter to move to the next input or submit the form, " + "\n" +
 		"up and down arrows to scroll the view, Ctrl+C twice to quit):" + "\n\n\n")
 
-	for i := range m.textInputs {
-		b.WriteString(m.prompts[i])
+	for i := range m.inputs {
+		input := m.inputs[i]
```

Comment:
:bulb: 

```suggestion
	for i, input := range m.inputs {
```
There's a few other for loops that can be simplified by just grabbing the second argument from range.

Reply:
Good catch! JS habits die hard. 🤣

Reply:
Reworked the loops.